models/issues/issue.go

```diff
@@ -1663,28 +1701,41 @@ func getIssueStatsChunk(opts *IssueStatsOptions, issueIDs []int64) (*IssueStats,
 			sess.In("issue.id", issueIDs)
 		}
 
-		if len(opts.Labels) > 0 && opts.Labels != "0" {
-			labelIDs, err := base.StringsToInt64s(strings.Split(opts.Labels, ","))
+		labelIDs, err := base.StringsToInt64s(strings.Split(opts.Labels, ","))
+		if len(labelIDs) > 0 {
 			if err != nil {
 				log.Warn("Malformed Labels argument: %s", opts.Labels)
 			} else {
 				for i, labelID := range labelIDs {
 					if labelID > 0 {
 						sess.Join("INNER", fmt.Sprintf("issue_label il%d", i),
 							fmt.Sprintf("issue.id = il%[1]d.issue_id AND il%[1]d.label_id = %[2]d", i, labelID))
-					} else {
+					} else if labelID < 0 {
 						sess.Where("issue.id NOT IN (SELECT issue_id FROM issue_label WHERE label_id = ?)", -labelID)
 					}
 				}
 			}
 		}
 
-		if opts.MilestoneID > 0 {
-			sess.And("issue.milestone_id = ?", opts.MilestoneID)
+		var mileIDs, noMileIDs []int64
```

Comment:
we should use a xorm.Builder and not select by In(BigSlice)

Reply:
Maybe in 7137988ded1d4ad558992cc01068cc8f9f240080 fixed?